compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -108,6 +106,16 @@ class ComposeScene internal constructor(
         invalidate,
     )
 
+    private val childScenes = mutableSetOf<ComposeScene>()
+
+    fun addChildScene(scene: ComposeScene) {
+        childScenes.add(scene)
```

Comment:
```suggestion
        check(!isClosed) { "ComposeScene is closed" }
        childScenes.add(scene)
```

Reply:
I don't see a case where this could happen, but sure, I'll add it just in case.